girder/api/describe.py

```diff
@@ -259,15 +261,16 @@ def modelParam(self, name, description=None, model=None, destName=None, paramTyp
         and the model should be loaded and passed into the route handler. For example,
         if you have a route like ``GET /item/:id``, you could do:
 
-        >>> modelParam('id', model='item', level=AccessType.READ)
+        >>> from girder.models.item import Item
+        >>> modelParam('id', model=Item, level=AccessType.READ)
 
         Which would cause the ``id`` parameter in the path to be mapped to an
         item model parameter named ``item``, and ensure that the calling user
         has at least ``READ`` access on that item. For parameters passed in
         the query string or form data, for example a request like
         ``POST /item?folderId=...``, you must specify the ``paramType``.
 
-        >>> modelParam('folderId', 'The ID of the parent folder.',
+        >>> modelParam('folderId', 'The ID of the parent folder.', model=Folder,
         ...            level=AccessType.WRITE, paramType='query')
 
         Note that in the above example, ``model`` is omitted; in this case, the
```

Comment:
This PR added a `model` parameter to the above example, so this statement is confusing.